kuma/wiki/tests/conftest.py

```diff
@@ -98,6 +98,20 @@ def trans_edit_revision(trans_doc, edit_revision, wiki_user):
     return trans_doc.current_revision
 
 
+@pytest.fixture
+def redirect_doc(wiki_user):
+    """A newly-created top-level English redirect document."""
+    redirect_doc = Document.objects.create(
+        locale='en-US', slug='Redirection', title='Redirect Document')
+    Revision.objects.create(
+        document=redirect_doc,
+        creator=wiki_user,
+        content='REDIRECT <a class="redirect" href="/blah">Some Redirect</a>',
```

Comment:
I think this will be a more interesting fixture for future work if it was a redirect to the ``root_doc``. If we do that, it could throw off the document count for ``test_list_no_redirects``.